App.tsx

```diff
@@ -65,11 +65,11 @@ class App extends Component<{}, appState> {
     };
   }
 
-  setMapLocation = (location: Location) => {
+  setMapLocation = (latitude, longitude) => {
     this.setState({
       region: {
-        latitude: location.getLatitude(),
-        longitude: location.getLongitude(),
+        latitude,
+        longitude,
```

Comment:
Why was this changed, location object should be the standard

Reply:
I have no idea, Ill add it back. I did not touch this file ;.<